scripts/ssb-web/install

```diff
@@ -18,6 +18,15 @@ sudo npm install -g --unsafe-perm \
 # Define location of SSB installation
 ssbPath="$HOME/.ssb"
 
+# Install broadcast service for users of sbot
+currentUser=$USER
+sudo cp "$BASE_DIR/ssb-web-broadcast-service.sh" "/usr/local/bin/ssb-web-broadcast-service.sh"
+sudo cp "$BASE_DIR/ssb-web-broadcast.service" /etc/systemd/system/ssb-web-broadcast.service
+sudo sed -i "s|_USER_|${currentUser}|" /etc/systemd/system/ssb-web-broadcast.service
```

Comment:
We should probably use `__USER__` because it's easy to unintentionally replace something like `MY_USER_NAME` when `_USER_` is the search string.

Also do `|g` always to avoid future headaches.